api/accounts/accounts_test.go

```diff
@@ -578,45 +576,3 @@ func batchCallWithNonExistingRevision(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, statusCode, "bad revision")
 	assert.Equal(t, "revision: leveldb: not found\n", string(res), "revision not found")
 }
-
-func httpPost(t *testing.T, url string, body interface{}) ([]byte, int) {
```

Comment:
why we remove all these methods ? are they somewhere else or we don't use them at all ?

Reply:
They're not used any longer. I think they were made redundant by the thor client merge.